test/vanilla/AcceptanceTests/test_xml.py

```diff
@@ -183,3 +183,8 @@ def test_models(self):
         else:
             from xmlservice.models._models import Error as ErrorPy2
             assert Error == ErrorPy2
+
+    def test_xms_text(self, client):
```

Comment:
After https://github.com/Azure/autorest.python/pull/725 is merged, will add async xms-text test as well